### PR TITLE
Mobile web: Add aria-labels to navbar buttons #13010

### PR DESCRIPTION
**Pull request by MathewtheCoder**
### Summary
Added **aria-label** to both info button and search button in mobile web view.

#### Ticket Link
 Fixes https://github.com/mattermost/mattermost-server/issues/13010
JIRA: https://mattermost.atlassian.net/browse/MM-19752

### CODE REVIEW

File: components/channel_header_mobile/channel_info_button/channel_info_button.js [hunk 1]
```diff
@@ -118,7 +118,10 @@ export default class NavbarInfoButton extends React.PureComponent {
                 className='description'
                 rootClose={true}
             >
-                <button className='navbar-toggle navbar-right__icon navbar-info-button pull-right'>
+                <button
+                    className='navbar-toggle navbar-right__icon navbar-info-button pull-right'
+                    aria-label='Info'
```
Comment (hmhealey) [inline components/channel_header_mobile/channel_info_button/channel_info_button.js]: This one should be translatable

File: components/channel_header_mobile/show_search_button/show_search_button.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ export default class ShowSearchButton extends React.PureComponent {
                 type='button'
                 className='navbar-toggle navbar-right__icon navbar-search pull-right'
                 onClick={this.handleClick}
+                aria-label='Search'
```
Comment (hmhealey) [inline components/channel_header_mobile/show_search_button/show_search_button.js]: And this one as well

**Comment by MathewtheCoder** [comment]
@saturninoabril @hmhealey I have done the translations as requested.

**Pull request merged**